Automattic/wp-calypso
# try: add storybook to showcase progressbar from calypso-ui

**WIP**

**Note**: At the moment this is just an exploration to see whether it's possible to use Storybook to showcase our components. Also this is based on #33154 to be able to import `<ProgressBar />` from `@automattic/calypso-ui`.

#### Testing instructions

First run

```
npm run distclean && npm ci
```

And then `npm run storybook`. Here's what you should see:

![localhost_6006__path=_story_progressbar--pulsing](https://user-images.githubusercontent.com/9202899/58025783-793bc280-7b15-11e9-93e2-2f891ca5950f.png)

cc @blowery
